AppCenterDistribute/AppCenterDistribute/MSACDistribute.m

```diff
@@ -449,6 +449,16 @@ - (void)requestInstallInformationWith:(NSString *)releaseHash {
   }
 }
 
+- (void)checkDelegateAndInvokeOnNoReleaseAvailableCallback {
+  id<MSACDistributeDelegate> delegate = self.delegate;
+  if ([delegate respondsToSelector:@selector(onNoReleaseAvailableonNoReleaseAvailable)]) {
```

Comment:
`onNoReleaseAvailableonNoReleaseAvailable` Is it typo?

Reply:
yup, it is 🤦‍♂️ Thanks